lib/measures/urban_geometry_creation_zoning/measure.rb

```diff
@@ -766,7 +222,7 @@ def run(model, runner, user_arguments)
           
           # get first floor footprint points
           other_building_points = []
-          multi_polygons = get_multi_polygons(other_building)
+          multi_polygons = geojson_gem.get_multi_polygons(other_building)
```

Comment:
In general, I would prefer things that are free functions to just be free functions and not have to make an instance of the extension to run them.  You could just put these in the URBANopt::GeoJSON module